File: rb/test/exists_test.rb
# RealTimeBusData SDK exists test

require "minitest/autorun"
require_relative "../RealTimeBusData_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = RealTimeBusDataSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: lua/test/exists_test.lua
-- RealTimeBusData SDK exists test

local sdk = require("real-time-bus-data_sdk")

describe("RealTimeBusDataSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
